frontend/packages/dev-console/src/components/topology/bindable-services/models.ts

```diff
@@ -0,0 +1,13 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const BindableServicesModel: K8sKind = {
+  apiGroup: 'binding.operators.coreos.com',
+  apiVersion: 'v1alpha1',
+  kind: 'BindableService',
+  plural: 'BindableServices',
+  label: 'BindableService',
+  labelPlural: 'BindableServices',
```

Comment:
```suggestion
  labelPlural: 'Bindable Services',
```

Reply:
@sahil143 @rottencandy 
The latest localization uses the same PascalCasing as the kind in the label. Have a look at https://github.com/openshift/console/blob/master/frontend/public/models/index.ts

Reply:
Reverted `label` & `labelPlural` to PascalCase.